modules/packages.xqm

```diff
@@ -282,11 +282,13 @@ declare function packages:get-package-meta($app as xs:string, $name as xs:string
 (: should be private but there seems to be a bug :)
 declare function packages:public-repo-contents($installed as element(repo-app)*) {
     try {
-        let $url := $config:DEFAULT-REPO || "/public/apps.xml?version=" || packages:get-version() ||
+        for $pkgs in $packages:repos
+        let $url := $pkgs || "/public/apps.xml?version=" || packages:get-version() ||
```

Comment:
plural? urls?

Reply:
$url is one string? it is a simple single string concatenation no?

Reply:
it's a sequence of strings